ymca/src/main/webapp/js/LoggingEndpoint.js

```diff
@@ -14,11 +14,21 @@ var loggingEndpoint = {
 			
 			websocket.onmessage = function(message) 
 			{
-				var jsonMessage = JSON.parse(message.data);
-				console.log(jsonMessage['level'] + ": " + jsonMessage['message']);
+				var jsonMessage = JSON.parse(message.data);				
+				loggingEndpoint.log(jsonMessage.level, jsonMessage.message);
 			};
 		},
 		
+		log(level, message)
```

Comment:
if we want a consistant code style this should be `log : function(level, message)` same for `sendId()` method below.